--- confparse.py.orig	2024-08-21 15:42:36.323363173 +0000
+++ confparse.py	2024-08-21 15:42:57.186146136 +0000
@@ -17,7 +17,7 @@
 '''Get values of the configuration file'''
 
 # standard library imports
-from configparser import SafeConfigParser
+from configparser import ConfigParser
 import logging
 import os
 import os.path
@@ -58,7 +58,7 @@
         for pathtoconfig in self.clioptions.configs:
             options = {}
             # read the configuration file
-            config = SafeConfigParser()
+            config = ConfigParser()
             if not config.read(os.path.expanduser(pathtoconfig)):
                 sys.exit('Could not read config file')
             ####################
